### DISCUSSION TITLE
Contributor Call: 1 October 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2828] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 1 October 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/oct-1-2022/4pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.

### Agenda
TBD

> Notes will be added after the call.

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark 
- @KaerMorhan 
- @bobgeorgethe3rd 
- @lexander2002 
- @BenJamesBen 
- @starfetch 
- @SeaZeeZee 
- @joostdecock 
- @nchilada 
- @woutervdub 
- @eriese 

✨ **Zwaluw**
Documentation for Jane is mostly finished. It’s almost ready to get published. During the instructions a question appeared: what kind of audience it should be written for? For a total beginner or for a more experienced sewer?
_Karen: I usually assume that the reader knows the basics of sewing and knows how their sewing machine works but all of the other details I mention.
Joost: we have a lot of patterns so instructions need to be written specifically for the pattern, questions about general sewing can be linked to sewing documentation.
Eriese: also if there's general sewing documentation that we should have, that's a worthwhile suggestion, too._

✨ **Lexander**
Maybe new Discord forum feature can be implemented into FS server for more structured question answering. It might be tricky to implement it (name, how it will work).
_Joost: we should probably try it and see how it works and adapt to the results afterwards.
Eriese: maybe it would be useful to introduce a bot that would work in a “it looks like you’re asking this thing that is in the FAQ, here’s the link” way._

✨ **Starf**
Labels on GitHub, it looks like they have been duplicated. Is there a reason for it? Which ones should we use now? Can I delete old labels with the package?
_Nchilada: it was an accident, sorry (issue #2619).
Joost: packages are related to history of how things on FS were in the past, it can be changed.
Bob: if the bot is working with the new labels, the old labels should probably be deleted.
Joost: we probably should remember that labels will probably be more searched than the emoji._

✨ **Joost**
Some of the pull requests and new issues weren’t answered. Some things will probably get worse at first before it gets better while we’re setting on a new way of working.

_Eriese: should we have a pre-commit hook when we have 600+ issues that don’t get fixed.
Joost: I didn’t think about it. We could disable the pre-commit hook. Committing changes is hard because they don’t pass the linter and a lot of mistakes are showing. Merge-develop into your own pathway has some problems._

_Zwaluw: can beginner coders help with code cleaning?
Eriese: yes, we can create a guide._

_Nchilada: it might be better to take this task off of Joost’s shoulders. I can create pull-requests for disabling the hook and fixing some of the mistakes in the code._

_Benjamin: please notify ahead of the time about any changes like enabling the pre-commit hook because it was unexpected and started affecting commits in a way that I didn’t understand._

_Benjamin: maybe it might be a good idea to bring back a developer call so that coders can stay on the same page and discuss stuff in depth that is not possible during the CC with people who are not well-versed in IT terminology.
Karen: it is also appropriate to bring up developer issues on the CC.
Eriese: there could be a chat or a thread where developers can update each other on the stuff they’re working on._

- (A thread was created, [What I'm working on](https://discord.com/channels/698854858052075530/1025817878068469872)).

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
